pkg/packages.go

```diff
@@ -200,16 +204,26 @@ func checkLegacyNameTaken(legacyName string, pkgName string) (bool, error) {
 }
 
 func known(deps map[string]deps.Dependency, p string) bool {
-	p = filepath.ToSlash(p)
+	p = Subdir(filepath.ToSlash(p))
 	for _, d := range deps {
-		k := d.Name()
+		k := Subdir(d.Name())
```

Comment:
```suggestion
		k := filepath.ToSlash(d.Name())
```
This change alone would suffice to fix the issue.

In your change, every `/` in `p` is replaced by `\` and the same for `d.Name()`. Using `filepath.ToSlash` we can eliminate the need for the `Subdir` function

Reply:
Ah okay, I have just pushed the updated code. Let me know if there is anything i miss